### ISSUE TITLE
what is the filename "BAZA_DANYCH.xlsx" when I invoked method "paths.get_data('bdi', study='C')"

### ISSUE THREAD

**Issue by Aknifejackzhmolong** [mmagnuski/DiamSar#29]
I had downloaded dataset DiamSar, Wronski and Nowowiejska, but when I invoked method "paths.get_data('bdi', study='C')" it printed a Exception "BAZA_DANYCH.xlsx not found". So it is bugs? Or any my mistakes?

The file under  the path "path to base_dir/DiamSar/beh" just a "beh_preproc.csv", I try to rename it&save it as excel format but it only work in study C.

**Comment by mmagnuski** [comment]
Hi @Aknifejackzhmolong,
sorry for responding after almost 1 year, but unfortunatelly I completely missed that an issue was created here.
I will take a look at this, but it might be working now. When we were publishing the data at Dryad we had to adapt the paticipant info databases to not reveal to much of personal data (Dryad asked us to keep the subject information below 4 variables). A few months after these changes I noticed that commits that kept DiamSar repo in up to date with these modifications were stuck on my laptop and never commited to github. These commits are already in DiamSar.
